proto/sanitizer.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+package pluginextensionv2;
+
+option go_package = ".;pluginextensionv2";
+
+message SanitizeRequest {
+  string filename = 1;
+  bytes content = 2;
+  bytes domPurifyConfig = 3;
+  bool allowAllLinksInSvgUseTags = 4;
+}
+
+message SanitizeResponse {
+  string error = 1;
+  bytes sanitized = 2;
```

Comment:
is there relevant info we may also return?  I guess we can add that later

Reply:
not sure, I could add a generic `string info;` field and for now populate it with "sanitized" / "not sanitized" depending on whether the hash changed. We could use that field in the future if we choose to migrate to some other library that returns some info on itself.

Reply:
it would be difficult to differentiate between sanitized vs unsanitized files even right now, as dompurify modifies all svgs. I think we can do it later if we'd like